cpp/include/cudf/io/nvcomp_adapter.hpp

```diff
@@ -36,33 +36,20 @@ struct feature_status_parameters {
   int lib_patch_version;                 ///< patch version
   bool are_all_integrations_enabled;     ///< all integrations
   bool are_stable_integrations_enabled;  ///< stable integrations
-  int compute_capability_major;          ///< cuda compute major version
 
   /**
-   * @brief Default Constructor
+   * @brief Default Constructor, with the current version of nvcomp and current environment
+   * variables
    */
   feature_status_parameters();
 
   /**
-   * @brief feature_status_parameters Constructor
+   * @brief feature_status_parameters Constructor, with the current version of nvcomp
```

Comment:
I think I'm misreading this:  

> `with the current version of nvcomp`

The constructor no longer takes major/minor versions.  So this phrase means "for the current version of nvcomp"?

Reply:
looks like I worded this poorly. What I'm trying to say is that the version of nvcomp that we built with will be used to populate the version fields. This will change over time without code changes, hence "current".